tests/Types/ArrayTest.php

```diff
@@ -21,7 +24,9 @@ class ArrayTest extends TestCase
     protected function setUp(): void
     {
         $this->platform = $this->createMock(AbstractPlatform::class);
-        $this->type     = new ArrayType();
+        $type           = Type::getType(Types::ARRAY);
```

Comment:
This is a step backward, IMO. Testing a type shouldn't require the usage of a type registry.